### PR TITLE
Update to the SDK Release process

### PR DESCRIPTION
**Pull request by viorel15**
- The release process contains link to internal process to raise a CR
- The SDK Release has been decoupled form the App release

### CODE REVIEW

File: release.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-
+
```
Comment (viorel15) [inline release.md]:
```suggestion
```

File: release.md [hunk 2]
```diff
@@ -1,4 +1,3 @@
-
 ## Release process
```
Comment (viorel15) [inline release.md]:
```suggestion

## Release process
```

File: release.md [hunk 3]
```diff
@@ -55,13 +55,16 @@
...
-	1. Ask SDK team (#sdk_squad) about the SDK version number
-	1. Trigger a hockey build from that branch using its command (eg. `/distribute_sdk version:0.1.11 pre_release:false branch:release/3.7.0`) in `#ios-build`.
 1. Merge the changes back to develop.
 
+### 3. SDK Release
 
-### 3. Release calendar
+1. Ask SDK team (#sdk_squad) about the SDK version number.
+2. Follow the [Internal SDK Release Process](https://engineering.ops.babylontech.co.uk/docs/cicd-deployments/#mobile-sdk-releases-ios-android) to open a CR ticket on the CRP board.
+3. Trigger a hockey build from that branch using its command (eg. `/distribute_sdk version:0.1.11 pre_release:false branch:release/3.7.0`) in `#ios-build`.
```
Comment (mr-v) [inline release.md]: `pre_release` argument is no longer needed. It's determined now based on `version`, e.g. `0.1.11` is release and `0.1.11-beta` is prerelease.

Comment (viorel15) [reply release.md]: I will update the command as soon as we remove the legacy. I think you have created a story for that

Comment (RuiAAPeres) [reply release.md]: Any update on this?

Comment (viorel15) [reply release.md]: The command and process has been udpated

**Review by mr-v — APPROVED** [review]

File: release.md [hunk 4]
```diff
@@ -55,13 +55,18 @@
...
-	1. Trigger a hockey build from that branch using its command (eg. `/distribute_sdk version:0.1.11 pre_release:false branch:release/3.7.0`) in `#ios-build`.
 1. Merge the changes back to develop.
 
+### 3. SDK Release
 
-### 3. Release calendar
+1. Ask SDK team (#sdk_squad) about the SDK version number.
+2. Follow the [Internal SDK Release Process](https://engineering.ops.babylontech.co.uk/docs/cicd-deployments/#mobile-sdk-releases-ios-android) to open a CR ticket on the CRP board.
+3. Cut a release branch for the SDK from the app release branch (eg. `sdk/0.5.0`)
+4. Create PR and udpate the SDK changelog `babylon-ios-release/SDK/CHANGELOG.md` to add the release version and date
```
Comment (ilyapuchka) [inline release.md]: `babylon-ios-release/SDK/CHANGELOG.md` what is this path?

File: release.md [hunk 5]
```diff
@@ -55,13 +55,18 @@
...
+5. Trigger a hockey build from that branch using its command (eg. `/fastlane distribute_sdk_v2 version:0.5.0 branch:release/sdk/0.5.0`) in `#ios-build`.
```
Comment (ilyapuchka) [inline release.md]: do we still need `v2` or we can get rid of first version of the lane?

Comment (viorel15) [reply release.md]: We still need this since the old distribution has not been removed. There is a PR for that

File: release.md (hunk 4 above)
Comment (viorel15) [reply release.md]: My fault. Thank you for spotting that.
 I have updated that to `SDK/CHANGELOG.md`

**Comment by RuiAAPeres** [comment]
Can you fix the conflicts so we can merge this @viorel15 ?

**Comment by viorel15** [comment]
done

**Comment by mr-v** [comment]
Another review is required.

**Review by JasonDobo — APPROVED** [review]

**Pull request merged**